This just paves the way to calculate more useful defaults in a shell.
---
 check/Makefile.am | 1 -
 configure.ac      | 4 ++++
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/check/Makefile.am b/check/Makefile.am
index 30a527c..5006b5f 100644
--- a/check/Makefile.am
+++ b/check/Makefile.am
@@ -1,4 +1,3 @@
-TESTS_PKG_CONFIG = $(top_builddir)/pkg-config$(EXEEXT)
 TESTS_ENVIRONMENT = PKG_CONFIG='$(TESTS_PKG_CONFIG)' $(TESTS_SHELL)
 
 TESTS = \
diff --git a/configure.ac b/configure.ac
index 4de9b16..73f2cf4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -32,6 +32,10 @@ AC_ARG_VAR([TESTS_SHELL], [Path to a POSIX shell to be used for testing])
 conf_path="$PATH:`getconf PATH 2>/dev/null`"
 AC_PATH_PROGS([TESTS_SHELL], [bash ksh sh], [$CONFIG_SHELL], [$conf_path])
 
+dnl The built pkg-config to use for testing. This allows it to be set
+dnl from the make command line if necessary.
+AC_SUBST([TESTS_PKG_CONFIG], ['${top_builddir}/pkg-config${EXEEXT}'])
+
 dnl
 dnl Default pkg-config search path
 dnl
-- 1.8.1.4 